pulser/_seq_drawer.py

```diff
@@ -199,6 +202,34 @@ def phase_str(phi):
         a.set_ylabel(r'$\Omega$ (rad/µs)', fontsize=14, labelpad=10)
         b.set_ylabel(r'$\delta$ (rad/µs)', fontsize=14)
 
+        if draw_phase_area:
+            top = False  # Variable to track position of box, top or center.
+            for pulse_num, seq_ in enumerate(seq._schedule[ch]):
+                # Select only `Pulse` objects
+                if isinstance(seq_.type, Pulse):
+                    phase_val = seq_.type.phase / np.pi
+                    area_val = seq_.type.amplitude.integral / np.pi
+                    x_plot = (seq_.ti + seq_.tf) / 2
```

Comment:
Be careful, this needs to be adjusted for the `time_scale` (it changes to µs when the total duration reaches 10 µs). Try to make a long enough sequence and check if the boxes are still placed correctly.

Reply:
Thanks, will check this..